Solutions/AliterConsultingSoD/Watchlists/AliterSODRisks.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+      "workspaceName": {
+          "type": "string",
+          "metadata": {
+              "description": "Workspace name for Log Analytics where Sentinel is setup"
+          }
+      },
+        "watchlistdescription": {
+            "type": "string",
+            "metadata": {
+                "description": "Risks for SOD Reporting"
+            }
+        }
+  },
+    "resources": [
+        {
+        "name": "[concat(parameters('workspaceName'), '/Microsoft.SecurityInsights/ReferenceTemplate')]",
+        "type": "Microsoft.OperationalInsights/workspaces/providers/Watchlists",
+        "kind": "",
+        "properties": {
+            "displayName": "ReferenceTemplate",
+            "source": "AliterSODFunctions.csv",

```

Comment:
Hi @gajackson1963 I had a little confusion on this. Both the watchlists are having same displayName and source but different rawContent. While deploying instead of creating two different instances it is updating the watchlist. Is it an expected behaviour?